admin_guide/upgrading_citus.rst

```diff
@@ -88,22 +88,26 @@ After installing the new package and restarting the database, run the extension
 
 .. _upgrading_postgres:
 
-Upgrading PostgreSQL version from 9.6 to 10.0
-#############################################
+Upgrading PostgreSQL version from 10 to 11
+##########################################
+
+.. note::
+
+   Do not attempt to upgrade *both* Citus and Postgres versions at once. If both upgrades are desired, start with a database upgrade first.
```

Comment:
This is a bit ambiguous. What do we mean by 'database upgrade'? I think we should note that 7.5 *does not* work with PG11. So any update to 11 *requires* a Citus major version upgrade to 8.0. Only then can we upgrade to PG11.